chall02/dalba-de.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+dictionary = {"a": ".-", "b": "-...", "c": "-.-.", "d": "-..", "e": ".", "f": "..-.", 
+    "g": "--.", "h": "....", "i": "..", "j": "·---", "k": "-.-", "l": ".-..", 
+    "m": "--", "n": "-.", "o": "---", "p": ".--.", "q": "--.-",
+    "r": ".-.", "s": "...", "t": "-", "u": "..-", "v": "...-", "w": ".--",
+    "x": "-..-", "y": "-.--", "z": "--..",
+    "A": ".-", "B": "-...", "C": "-.-.", "D": "-..", "E": ".", "F": "..-.", 
+    "G": "--.", "H": "....", "I": "..", "J": "·---", "K": "-.-", "L": ".-..", 
+    "M": "--", "N": "-.", "O": "---", "P": ".--.", "Q": "--.-",
+    "R": ".-.", "S": "...", "T": "-", "U": "..-", "V": "...-", "W": ".--",
+    "X": "-..-", "Y": "-.--", "Z": "--.."}
+
+import sys
+
+texto = ""
+
+if len(sys.argv) == 2:
+    if len(sys.argv[1]) == 0:
+        print "usage: ./dalba-de.py <a-zA-Z string>"

```

Comment:
You should make this message more dynamic
```suggestion
        print("usage: %s <a-zA-Z string>" % sys.argv[0])
```
Note that I added parentheses, it is normal, you were supposed to turn-in your code in `python3.x` your turned it in in `python2.7` [which is deprecated](https://pythonclock.org/) since January the 1st of 2020.